src/System.Net.Sockets/src/System/Net/Sockets/IOControlKeepAlive.Windows.cs

```diff
@@ -0,0 +1,98 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Net.Sockets
+{
+    internal class IOControlKeepAlive
```

Comment:
Nit: can you add a comment to this class about why it's needed?  That'll help us to remove it in the (far) future when such platforms are no longer supported.